packages/store/src/lib/actions.ts

```diff
@@ -0,0 +1,11 @@
+export enum EntityActions {
```

Comment:
The base store doesn't need to know anything about entities.

Reply:
I've changed the name to `EntityActions` to ease the migration from Akita. I guess that's better, but I can change it back. let me know...

Reply:
EntityActions should be in the entities package